src/Collections-Weak/WeakValueDictionary.class.st

```diff
@@ -71,6 +71,15 @@ WeakValueDictionary >> at: key put: anObject [
 	^ anObject
 ]
 
+{ #category : #testing }
+WeakValueDictionary >> includesKey: key [
+	"Answer whether the receiver has a key equal to the argument and also that the value associated to this key was not garbage collected."
+
+	^ (array at: (self scanFor: key))
+		ifNil: [ false ]
+		ifNotNil: [ :value | value value isNotNil ]
```

Comment:
unrelated comment:
Do we really decide to move to #isNotNil methods instead of #notNil? For me the consistency is to use the only version and I doubt that it is possible to deprecate #notNil. Then we will always have both versions in the image.

Reply:
Personally, I always use #isNotNil for consistency with #isNil. I prefer to have the same prefix in all cases. I do not find consistent to have `notNil` and `isNil`.

Reply:
I understand the original intention of #isNotNil. And I would also prefer this kind of consistency for testing methods where all of them are starting with #is prefix. 
But the reality is that we can't remove notNil and notEmpty (do you agree?). And then I would prefer couple of exceptions from the convention either than duplicated methods.